dev/com.ibm.ws.jbatch.open_fat/publish/servers/checkpointbatchFAT/bootstrap.properties

```diff
@@ -0,0 +1,13 @@
+com.ibm.ws.logging.trace.specification=*=info=enabled\
+:test=all=enabled\
+:com.ibm.jbatch.*=all=enabled\
+:jsr352.test.servlet.*=all=enabled\
+:com.ibm.ws.jbatch.*=all=enabled\
+:BonusPayout=all=enabled\
+:com.ibm.ws.jbatch.open_fat=all=enabled\
+:batch.fat.*=all=enabled\:checkpoint=all
+
+io.openliberty.checkpoint.allowed.features=batch-1.0,batch-2.0,batch-2.1,batch-management-1.0
+
+bootstrap.include=../testports.properties
+osgi.console=5678
```

Comment:
I'm surprised this one doesn't have:
```
websphere.java.security.exempt=true
```

Reply:
I do think we need this `websphere.java.security.exempt=true` added to the `bootstrap.properties` for this server config used for checkpoint.